data/json/recipes/recipe_robots.json

```diff
@@ -0,0 +1,218 @@
+[
+  {
+    "type": "recipe",
+    "result": "hp_compressor",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "mechanics",
+    "skills_required": [ [ "electronics", 1 ], [ "fabrication", 1 ] ],
+    "difficulty": 3,
+    "time": 20000,
+    "autolearn": true,
+    "reversible" : true,
+    "using": [ [ "welding_standard", 2 ], [ "soldering_standard", 2 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "WRENCH", "level": 1 } ],
+    "components": [
+      [ [ "pump_complex", 1 ] ],
+      [ [ "power_supply", 1 ] ],
+      [ [ "motor_small", 1 ] ],
+      [ [ "drivebelt", 1 ], [ "drivebelt_makeshift", 1 ] ],
+      [ [ "cable", 2 ] ],
+      [ [ "scrap", 3 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "hydraulic_platform",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "mechanics", 4 ] ],
+    "difficulty": 5,
+    "time": 360000,
+    "autolearn": true,
+    "reversible" : true,
+    "using": [ [ "welding_standard", 15 ] ],
+    "qualities": [
+      { "id": "SCREW", "level": 1 },
+      { "id": "WRENCH", "level": 2 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "HAMMER", "level": 1 },
+      { "id": "CUT", "level": 1 }
+    ],
+    "components": [
+      [ [ "frame", 1 ] ],
+      [ [ "jack", 2 ] ],
+      [ [ "hp_compressor", 1 ] ],
+      [ [ "hose", 2 ] ],
+      [ [ "spring", 4 ] ],
+      [ [ "plastic_chunk", 4 ] ],
+      [ [ "steel_lump", 1 ], [ "steel_chunk", 4 ], [ "scrap", 16 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "bot_navigation",
```

Comment:
Craftable full AI? I won't believe it.

Reply:
Sorry, it's not an AI at all, I'll give it more correct name. It's just simple guidance unit, only to be enough to follow player character.